### PR TITLE
Docs: fix grammar issues

### PR DESCRIPTION
**Pull request by alanisaac**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Small documentation changes to improve readability.

**Which issue(s) this PR fixes**:

Part of #2196 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/setup/upgrade.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Upgradation
+# Upgrading
```
Comment (daixiang0) [inline docs/setup/upgrade.md:1]: why not use nouns as titles?

Comment (alanisaac) [reply docs/setup/upgrade.md:1]: Upgradation isn't typically considered a word in several dialects of English.  See this Stack Exchange post: https://english.stackexchange.com/questions/192187/upgradation-not-universally-accepted

Just to check, I also took a look at other CNCF projects' documentation on the subject, and most that have upgrade documentation seem to follow the title pattern of 'Upgrading X'.  Here's some of what I found:
- [Argo](https://argoproj.github.io/argo-cd/operator-manual/upgrading/overview/) - uses 'upgrading' in the navigation and URL
- [Contour](https://projectcontour.io/resources/upgrading/) - 'Upgrading Contour'
- [Kubernetes](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) 'Upgrading kubeadm clusters'
- [Linkerd](https://linkerd.io/2/tasks/upgrade/) - 'Upgrading Linkerd'
- [NATS](https://docs.nats.io/nats-server/nats_admin/upgrading_cluster) - 'Upgrading a Cluster'
- [Vitess](https://vitess.io/docs/user-guides/upgrading-vitess/) - 'Upgrading Vitess'

Maybe this should be 'Upgrading KubeEdge'?

Comment (daixiang0) [reply docs/setup/upgrade.md:1]: 'Upgrading KubeEdge' sounds good, let's do it!
BTW, please update pr title like "Docs: fix grammar issues".

File: docs/components/edge/edged.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 ## Overview
 
-EdgeD is an edge node module which manages pod lifecycle. It helps user to deploy containerized workloads or applications at the edge node. Those workloads could perform any operation from simple telemetry data manipulation to analytics or ML inference and so on. Using `kubectl` command line interface at the cloud side, user can issue commands to launch the workloads.
+EdgeD is an edge node module which manages pod lifecycle. It helps users to deploy containerized workloads or applications at the edge node. Those workloads could perform any operation from simple telemetry data manipulation to analytics or ML inference and so on. Using `kubectl` command line interface at the cloud side, users can issue commands to launch the workloads.
 
-Docker container runtime is currently supported for container and image management. In future other runtime support shall be added, like containerd etc.,
+The Docker container runtime is currently supported for container and image management. In the future support for other runtimes shall be added, like containerd etc.,
```
Comment (GsssC) [inline docs/components/edge/edged.md:7]: @daixiang0 @alanisaac , CRI has already been supported by edgecore, how about delete these line?

File: docs/components/edge/edged.md [hunk 3]
```diff
@@ -48,8 +48,8 @@ This module helps in monitoring pod status for edged. Every second, using probe'
 Container Runtime Interface (CRI) – a plugin interface which enables edged to use a wide variety of container runtimes, without the need to recompile and also support multiple runtimes like docker, containerd, cri-o etc
 
 #### Why CRI for edge?
-Currently kubeedge edged supports only docker runtime using the legacy dockertools.
-+ CRI support for  multiple container runtime in kubeedge is needed due to below mentioned factors
+Currently KubeEdge edged supports only the docker runtime using legacy dockertools.
++ CRI support for multiple container runtimes in KubeEdge is needed due to the factors mentioned below
```
Comment (GsssC) [inline docs/components/edge/edged.md:52]: ditto, May need add comment to show that it has been realized.

File: docs/components/edge/edged.md [hunk 4]
```diff
@@ -66,58 +66,61 @@ Currently kubeedge edged supports only docker runtime using the legacy dockertoo
...
+The flow diagram below explains the message flow.
 
 ![Secret Message Handling](../../images/edged/secret-handling.png)
 
 *Fig 7: Secret Message Handling at EdgeD*
 
-Also edged uses MetaClient module to fetch secret from Metamanager (if available with it) else cloud. Whenever edged queries for a new secret which Metamanager doesn't has, the request is forwared to cloud. Before sending the response containing the secret, it stores a copy of it and send it to edged.
-Hence the subsequent query for same secret key will be responded by Metamanger only, hence reducing the response delay.
-Below flow diagram shows, how secret is fetched from metamanager and cloud. The flow of how secret is saved in metamanager.
+Also edged uses the MetaClient module to fetch secrets from Metamanager (if available), falling back to cloud. Whenever edged queries for a new secret which Metamanager doesn't have, the request is forwarded to cloud. Before sending the response containing the secret, it stores a copy of it and sends it to edged.
```
Comment (GsssC) [inline docs/components/edge/edged.md:77]: : ), what does `falling back to cloud` here mean?

File: docs/components/edge/edged.md (hunk 2 above)
Comment (daixiang0) [reply docs/components/edge/edged.md:7]: Please update it refer to [doc](https://docs.kubeedge.io/en/latest/configuration/cri.html).

File: docs/components/edge/edged.md (hunk 3 above)
Comment (fisherxu) [reply docs/components/edge/edged.md:52]: Agreed, the cri has been integrated at last year.

**Review by fisherxu — COMMENTED** [review]
Thanks for the great fixing for the docs! @alanisaac  Please just fix the comments and squash the commits, then let's got it merged. :)

File: docs/setup/upgrade.md (hunk 1 above)
Comment (alanisaac) [reply docs/setup/upgrade.md]: fixed doc title in 2edef69

Changed PR title to "Docs: fix grammar issues"

File: docs/components/edge/edged.md (hunk 3 above)
Comment (alanisaac) [reply docs/components/edge/edged.md]: fixed in 3db6642

File: docs/components/edge/edged.md (hunk 2 above)
Comment (alanisaac) [reply docs/components/edge/edged.md:7]: fixed in 3db6642
included a reference to the avoce linked documentation on how to configure for other runtimes

File: docs/components/edge/edged.md (hunk 4 above)
Comment (alanisaac) [reply docs/components/edge/edged.md]: @GsssC good question -- in my first pass I tried to keep the wording fairly similar to the original, but you're right this still isn't very clear.  I gave this paragraph and a similar paragraph on ConfigMaps another pass in f494caf based on the diagrams (which are very helpful!).

File: docs/components/edge/edged.md [hunk 5]
```diff
@@ -37,87 +37,80 @@ Its primary jobs are as follows:
...
+This module helps in monitoring pod status for edged. Every second, using probes for liveness and readiness, it updates the information with pod status manager for every pod.
 
 ![PLEG Design](../../images/edged/pleg-flow.png)
 
 *Fig 5: PLEG at EdgeD*
 
 ## CRI for edged
 
-Container Runtime Interface (CRI) – a plugin interface which enables edged to use a wide variety of container runtimes, without the need to recompile and also support multiple runtimes like docker, containerd, cri-o etc
+Container Runtime Interface (CRI) – a plugin interface which enables edged to use a wide variety of container runtimes like Docker, containerd, CRI-O, etc., without the need to recompile.  For more on how to configure KubeEdge for container runtimes, see [KubeEdge runtime configuration](../../configuration/cri.md).
```
Comment (daixiang0) [inline docs/components/edge/edged.md:48]: One extra blank, remove it.

Comment (alanisaac) [reply docs/components/edge/edged.md]: fixed in 051732b

File: docs/components/edge/edged.md [hunk 6]
```diff
@@ -37,87 +37,80 @@ Its primary jobs are as follows:
...
 
 ![Secret Message Handling](../../images/edged/secret-handling.png)
 
 *Fig 7: Secret Message Handling at EdgeD*
 
-Also edged uses MetaClient module to fetch secret from Metamanager (if available with it) else cloud. Whenever edged queries for a new secret which Metamanager doesn't has, the request is forwared to cloud. Before sending the response containing the secret, it stores a copy of it and send it to edged.
-Hence the subsequent query for same secret key will be responded by Metamanger only, hence reducing the response delay.
-Below flow diagram shows, how secret is fetched from metamanager and cloud. The flow of how secret is saved in metamanager.
+edged uses the MetaClient module to fetch secrets from MetaManager. If edged queries for a new secret which is not yet stored in MetaManager, the request is forwarded to the Cloud. Before sending the response containing the secret, MetaManager stores it in a local database. Subsequent queries for the same secret key will be retrieved from the database, reducing latency. The flow diagram below shows how a secret is fetched from MetaManager and the Cloud. It also descibes how the secret is stored in MetaManager.
```
Comment (kevin-wangzefeng) [inline docs/components/edge/edged.md:71]:
```suggestion
Edged uses the MetaClient module to fetch secrets from MetaManager. If edged queries for a new secret which is not yet stored in MetaManager, the request is forwarded to the Cloud. Before sending the response containing the secret, MetaManager stores it in a local database. Subsequent queries for the same secret key will be retrieved from the database, reducing latency. The flow diagram below shows how a secret is fetched from MetaManager and the Cloud. It also descibes how the secret is stored in MetaManager.
```

File: docs/components/edge/edged.md [hunk 7]
```diff
@@ -37,87 +37,80 @@ Its primary jobs are as follows:
...
 
 *Fig 9: ConfigMap Message Handling at EdgeD*
 
-Also edged uses MetaClient module to fetch configmap from Metamanager (if available with it) else cloud. Whenever edged queries for a new configmaps which Metamanager doesn't has, the request is forwared to cloud. Before sending the response containing the configmaps, it stores a copy of it and send it to edged.
-Hence the subsequent query for same configmaps key will be responded by Metamanger only, hence reducing the response delay.
-Below flow diagram shows, how configmaps is fetched from metamanager and cloud. The flow of how configmaps is saved in metamanager.
+edged uses the MetaClient module to fetch ConfigMaps from MetaManager. If edged queries for a new ConfigMap which is not yet stored in MetaManager, the request is forwarded to the Cloud. Before sending the response containing the ConfigMap, MetaManager stores it in a local database. Subsequent queries for the same ConfigMap key will be retrieved from the database, reducing latency. The flow diagram below shows how ConfigMaps are fetched from MetaManager and the Cloud. It also descibes how ConfigMaps are stored in MetaManager.
```
Comment (kevin-wangzefeng) [inline docs/components/edge/edged.md:92]:
```suggestion
Edged uses the MetaClient module to fetch ConfigMaps from MetaManager. If edged queries for a new ConfigMap which is not yet stored in MetaManager, the request is forwarded to the Cloud. Before sending the response containing the ConfigMap, MetaManager stores it in a local database. Subsequent queries for the same ConfigMap key will be retrieved from the database, reducing latency. The flow diagram below shows how ConfigMaps are fetched from MetaManager and the Cloud. It also descibes how ConfigMaps are stored in MetaManager.
```

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve

Thanks.

/hold
For mannual merging as this PR needs squash

**Pull request merged**